py_swf/clients/decision.py

```diff
@@ -157,7 +157,8 @@ def walk_execution_history(
 
             kwargs['nextPageToken'] = next_page_token
 
-    def finish_decision_with_activity(self, task_token, activity_id, activity_name, activity_version, activity_input):
+    def finish_decision_with_activity(self, task_token, activity_id, activity_name, activity_version, activity_input,
+                                      override_config_dict=None):
```

Comment:
Ah I was thinking passing whole config as a dict, but I think you are right we should use args here.